cmake/mcuboot.cmake

```diff
@@ -0,0 +1,183 @@
+# Included by mcuboot/zephyr/CMakeLists.txt
+set(MCUBOOT_DIR ${ZEPHYR_BASE}/../bootloader/mcuboot)
```

Comment:
ZEPHYR_MCUBOOT_MODULE_DIR ?

Reply:
No, it does not exist in this context actually